topics/reference_guide/custom_language_support/websymbols_integration.md

```diff
@@ -0,0 +1,309 @@
+
+The matched Web Symbol is taken as a scope for matching HTML attributes of the tag.
+
+#### Attributes {#html-attributes}
+
+|---|---|
+|||
+|Namespace|`html`|
+|Kind|`attributes`|
+
+Web Symbols, which represent available HTML attributes.
+HTML attribute symbols can be contributed statically or dynamically (through `WebSymbolsQueryConfigurator`).
+The statically contributed symbols are available globally, depending on the context setting of the contributing Web Types file.
+If the containing tag is matched to a Web Symbol it is added to the scope for attribute matching.
+
+For dynamic contributions the `WebSymbolsQueryConfigurator#getScope`'s `element` parameter can be:
+- `XmlAttribute` - Web Symbols should represent available HTML attributes for matching. *The HTML attribute actual name should not be taken into account when building the scope*, however the parent HTML tag and other attributes can be taken into account when building the scope.
+- `HtmlTag` - Web Symbols should represent available HTML attributes for matching. The HTML tag and other attributes can be taken into account when building the scope.
+
+Dedicated support for `WebSymbol` interface properties:
+- `required` - if `true`, warning will be shown if the attribute is missing. Does not apply to `virtual` attributes.
+- `default` - the default value of the attribute, if `attributeValue.default` is `null`.
+- `attributeValue` - provides information about the attribute value, see [`attributeValue`](websymbols_implementation.md#attributeValue) reference.
+
+### CSS
+
+#### Properties {#css-properties}
+
+|---|---|
+|||
+|Namespace|`css`|
+|Kind|`properties`|
+
+Web Symbols, which represent available CSS properties. [Custom CSS properties (variables)](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) names should be prefixed with `--`.
+CSS properties can be contributed statically or dynamically (through `WebSymbolsQueryConfigurator`).
+The statically contributed symbols are available globally, depending on the context setting of the contributing Web Types file.
+
+Within a CSS file, the additional scope for matching properties is built from Web Symbols matching HTML element names from terminal selectors from the enclosing ruleset.
+
+Within an HTML element `style` attribute value, the additional scope is built from Web Symbols matching the enclosing HTML element.
+
+Scope for custom properties (variables) references (arguments for [`var()`](https://developer.mozilla.org/en-US/docs/Web/CSS/var) function) is built the same way
+as for properties. Only properties with names starting with `--` are taken into account.
+
+For dynamic contributions the `WebSymbolsQueryConfigurator#getScope`'s `element` parameter can be:
+- `CssDeclaration` - Web Symbols should represent available CSS properties for matching. *The CSS declaration actual name should not be taken into account when building the scope*.
```

Comment:
remove italics

Reply:
I wanted to differentiate this from the rest of the paragraph. If you think it's better without itlics, I guess I need to remove them everywhere.